packages/vm/src/backends/cranelift.rs

```diff
@@ -1,14 +1,21 @@
 #![cfg(any(feature = "cranelift", feature = "default-cranelift"))]
 use wasmer_clif_backend::CraneliftCompiler;
-use wasmer_runtime_core::{backend::Compiler, compile_with, instance::Instance, module::Module};
+use wasmer_runtime_core::{
+    backend::Compiler, backend::CompilerConfig, compile_with_config, instance::Instance,
+    module::Module,
+};
 
 use crate::errors::{CompileErr, Error};
 use snafu::ResultExt;
 
 static FAKE_GAS_AVAILABLE: u64 = 1_000_000;
 
 pub fn compile(code: &[u8]) -> Result<Module, Error> {
-    compile_with(code, compiler().as_ref()).context(CompileErr {})
+    let config = CompilerConfig {
+        enable_verification: false, // As discussed in https://github.com/CosmWasm/cosmwasm/issues/155
```

Comment:
I know this is draft, but it would be interesting to flag this based on test or release, so it is false when we compile for tests and true when we compile for release.

You can do this via

```rust
#[cfg(debug_assertions)]
enable_verification: false,
..Default::default(),
```

[reference](https://doc.rust-lang.org/1.30.0/reference/conditional-compilation.html)

Reply:
I don't know that this cranelift IR (intermediate representation) verifier does, but I think it should not run un release builds.

From https://github.com/bytecodealliance/cranelift/issues/1247:

> Run the Cranelift IR verifier at strategic times during compilation. 
>
> This makes compilation slower but catches many bugs. [...]

Before https://github.com/wasmerio/wasmer/pull/1218, wasmer didabled it for us in all cases and unly enabled it when running Wasmer tests.

I think as long as we don't have a good reason, we should disable it.